src/main/java/org/sagebionetworks/bridge/sdk/BaseApiCaller.java

```diff
@@ -345,7 +328,7 @@ private void throwExceptionOnErrorStatus(HttpResponse response, String url) {
                 } else if (statusCode == 400 && message.contains("A published survey")) {
                     e = new PublishedSurveyException(message, url);
                 } else  if (statusCode == 400 && node.has("errors")) {
-                    Map<String, List<String>> errors = (Map<String, List<String>>) mapper.convertValue(
+                    Map<String, List<String>> errors = mapper.convertValue(
```

Comment:
This was marked as a warning in IntelliJ, which is why I changed it. Is the warning different in Eclipse?